docs/custom_domains.rst

```diff
@@ -1,85 +1,98 @@
 
-Using pip as an example, https://pip.pypa.io resolves, but is hosted on our infrastructure.
+   We don't currently support pointing subdomains or naked domains to a project using ``A`` records.
+   It's best to point a subdomain, ``docs.example.com`` for example, using a CNAME record.
 
-As another example, fabric's dig record looks like this::
+.. tabs::
 
-    -> dig docs.fabfile.org
-    ...
-    ;; ANSWER SECTION:
-    docs.fabfile.org.   7200    IN  CNAME   readthedocs.io.
+   .. tab:: Read the Docs Community
+      
+      - Add a CNAME record in your DNS that points the domain to ``readthedocs.io``
+      - Go the :guilabel:`Admin` tab of your project
+      - Click on :guilabel:`Domains`
+      - Enter your domain and click on :guilabel:`Add`
 
-Custom Domain SSL
------------------
+      By default, we provide a validated SSL certificate for the domain.
+      This service is generously provided by Cloudflare.
+      The SSL certificate issuance can take about one hour,
+      you can see the status of the certificate on the domain page in your project.
 
-By default, when you setup a custom domain to host documentation at Read the Docs,
-we will attempt to provision a domain validated SSL certificate for the domain.
-This service is generously provided by Cloudflare.
+      For example, https://pip.pypa.io resolves, but is hosted on our infrastructure.
+      As another example, fabric's dig record looks like this:
 
-After configuring your custom domain on Read the Docs,
-you can see the status of the certificate on the domain page in your project
-admin dashboard (:guilabel:`Domains` > :guilabel:`Edit Domain`).
+      .. prompt:: bash $, auto
 
-If your domain has configured CAA records, please do not forget to include
-Cloudflare CAA entries, see their `Certification Authority Authorization (CAA)
-FAQ <https://support.cloudflare.com/hc/en-us/articles/115000310832-Certification-Authority-Authorization-CAA-FAQ>`_.
+         $ dig docs.fabfile.org
+         ...
+         ;; ANSWER SECTION:
+         docs.fabfile.org.   7200    IN  CNAME   readthedocs.io.
 
-.. note::
+      .. note::
 
-    Some older setups configured a CNAME record pointing to ``readthedocs.org``
-    or another variation. While these continue to resolve,
-    they do not yet allow us to acquire SSL certificates for those domains.
-    Point the CNAME to ``readthedocs.io``, with no subdomain, and re-request a certificate
-    by saving the domain in the project admin (:guilabel:`Domains` >
-    :guilabel:`Edit Domain`).
+         Some older setups configured a CNAME record pointing to ``readthedocs.org`` or another variation.
+         While these continue to resolve,
+         they do not yet allow us to acquire SSL certificates for those domains.
+         Follow the new setup to have a SSL certificate.
 
-    If you change the CNAME record, the SSL certificate issuance can take about
-    one hour.
+      **Notes for Cloudflare users**
```

Comment:
This note would be better inside a `.. note::` directive. Or even better `.. admonition:: Cloudflare users`

Reply:
I think this is probably worth a `.. warning::`